Tella/Data/Networking/Repositories/ResourceRepository.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ResourceRepository.swift
+//  Tella
+//
+//  Created by gus valbuena on 2/7/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+struct ResourceRepository: WebRepository {
+    func getResourcesByProject(server: TellaServer) -> AnyPublisher<[ResourceDTO], APIError> {
```

Comment:
It's better to return a list of resource domain model instead of a list of `ResourceDTO`